snapcraft/internal/project_loader/_templates.py

```diff
@@ -121,6 +116,10 @@ def load_template(template_name: str) -> Dict[str, Any]:
 
 
 def _find_template(base: str, template_name: str) -> Dict[str, Any]:
+    # A bade is required in order to use templates, so raise an error if not specified.
```

Comment:
bade?

Reply:
Well duh. What would YOU call it? :wink: 

Fixed.